corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js

```diff
@@ -223,6 +223,7 @@ hqDefine('app_manager/js/forms/case_config_ui', function () {
         var caseConfigViewModel = function (caseConfig) {
             var self = {};
 
+            self.hasPrivilege = true;
```

Comment:
I was looking at the main [PR](https://github.com/dimagi/commcare-hq/pull/25004/files), where will this ever be set to `False`?

Reply:
in child cases. We have a privilege called `CHILD_CASES` that will be going into the new plans when Dev M is done with the final reallocation decisions. All the knockout templates in the case management part of the app are so tightly coupled though, that it makes this part a little tricky. This is for allowing to a project that has downgraded / lost the `CHILD_CASES` permission to see their configuration but not be able to edit anything related to it until they upgrade.

Reply:
Great, thanks!